tabs/ports.js

```diff
@@ -13,6 +13,7 @@ TABS.ports.initialize = function (callback, scrollPosition) {
          {name: 'TELEMETRY_FRSKY',      groups: ['telemetry'], sharableWith: ['msp'], notSharableWith: ['blackbox'], maxPorts: 1},
          {name: 'TELEMETRY_HOTT',       groups: ['telemetry'], sharableWith: ['msp'], notSharableWith: ['blackbox'], maxPorts: 1},
          {name: 'TELEMETRY_SMARTPORT',  groups: ['telemetry'], maxPorts: 1},
+         {name: 'TELEMETRY_ESC',        groups: ['telemetry'], maxPorts: 1},
```

Comment:
Hmm, I think this is wrong.

If you add `TELEMETRY_ESC` to the `telemetry` then it will appear in the same drop down as the telemetry protocols that can be used between FC and RX, and will make it impossible to configure ESC telemetry and 'normal' telemetry at the same time. I think what we need is a separate checkbox in the ports matrix for ESC telemetry.

Also, this needs to be version gated for >= 3.1 as well. In general, all new additions need feature gating, to prevent users with older firmware versions from seeing them.